Block/Adminhtml/Edit/Options/Options.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace SnowIO\AttributeOptionCode\Block\Adminhtml\Edit\Options;
+
+use Magento\Catalog\Setup\CategorySetup;
+use SnowIO\AttributeOptionCode\Api\AttributeOptionCodeRepositoryInterface;
+
+class Options extends \Magento\Eav\Block\Adminhtml\Attribute\Edit\Options\Options
+{
+    /** @var AttributeOptionCodeRepositoryInterface */
+    private $attributeOptionCodeRepository;
+
+    /**
+     * @var string
+     */
+    protected $_template = 'SnowIO_AttributeOptionCode::catalog/product/attribute/options.phtml';
+
+    /**
+     * Options constructor.
+     * @param \Magento\Backend\Block\Template\Context $context
+     * @param \Magento\Framework\Registry $registry
+     * @param \Magento\Eav\Model\ResourceModel\Entity\Attribute\Option\CollectionFactory $attrOptionCollectionFactory
+     * @param \Magento\Framework\Validator\UniversalFactory $universalFactory
+     * @param AttributeOptionCodeRepositoryInterface $attributeOptionCodeRepository
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Backend\Block\Template\Context $context,
+        \Magento\Framework\Registry $registry,
+        \Magento\Eav\Model\ResourceModel\Entity\Attribute\Option\CollectionFactory $attrOptionCollectionFactory,
+        \Magento\Framework\Validator\UniversalFactory $universalFactory,
+        AttributeOptionCodeRepositoryInterface $attributeOptionCodeRepository,
+        array $data = []
+    ) {
+        $this->attributeOptionCodeRepository = $attributeOptionCodeRepository;
+        parent::__construct($context, $registry, $attrOptionCollectionFactory, $universalFactory, $data);
+    }
+
+    /**
+     * @return array|null
+     */
+    public function getOptionValues()
+    {
+        $values = parent::getOptionValues();
```

Comment:
The return type here `@return array|null`, what happens if `$values` is `null` and is passed to `addOptionCodeToOptionValuesData`?